letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -19,7 +19,7 @@ XDG_DATA_HOME=${XDG_DATA_HOME:-~/.local/share}
 VENV_NAME="letsencrypt"
 VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN="$VENV_PATH/bin"
-LE_AUTO_VERSION="0.5.0.dev0"
+LE_AUTO_VERSION="0.4.1"
```

Comment:
Hmmm, now that I've merged this, we should probably also version bump this number in our release scripts  (though it doesn't so much matter until we land our next leauto change).